GrampsWebSync/const.py

```diff
@@ -1,30 +1,13 @@
 """Constants for Gramps Web Sync."""
 
+from __future__ import annotations
 
-class TypeMeta(type):
-    """Workaround for missing typing module in Gramps AIO."""
```

Comment:
@giotodibondone removing work around for Gramp AIO about **import typing** module as per https://gramps-project.org/bugs/view.php?id=12073 in case no one answers you...

@DavidMStraub  Love the Gramps Web Sync addon and Gramps web, Thanks 🐱‍🏍💯

Reply:
@PQYPLZXHGF Thanks , re-closed 12073 

@DavidMStraub Great work with the addon and gramps web.